drakrun/drakrun/lib/postprocessing/__init__.py

```diff
@@ -44,6 +45,17 @@ class PostprocessPlugin(NamedTuple):
         requires=["procmon.log"],
         generates=["process_tree.json"],
     ),
+    PostprocessPlugin(
+        function=capa_analysis,
+        requires=[
+            "apimon.log",
+            "syscall.log",
```

Comment:
I think that if `syscall.log` is available then that would be much better than `apimon.log` (with the default hooks).

Maybe I could remove both of these requirements and try to load each one in the capa plugin? This might make sense since neither of them is technically a requirement on its own, and the split_drakmon_log PostProcessing plugin entry doesn't list the files it generates?

Reply:
Yes, currently the plugin won't even run unless both logs are available (plugin requirements are checked automatically).

So I think both requirements should be removed from here, unfortunately.